### PR TITLE
Sriov feature gate

### PR DESCRIPTION
**Pull request by booxter**

### CODE REVIEW

**Comment by booxter** [comment]
@rmohr this is the documentation piece for the SRIOV feature gate you just landed.

File: workloads/virtual-machines/interfaces-and-networks.md [hunk 1]
```diff
@@ -337,6 +337,21 @@ passed through into the guest operating system as a host device, using the
...
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-config
+  namespace: kube-system
+  labels:
+    kubevirt.io: ""
+data:
+  feature-gates: "CPUManager"
```
[the remark this answers is not in the document]
Comment (booxter) [reply workloads/virtual-machines/interfaces-and-networks.md]: 🤦

File: workloads/virtual-machines/interfaces-and-networks.md [hunk 2]
```diff
@@ -347,14 +362,20 @@ to maintain high networking performance.
...
+    devices:
       interfaces:
         - name: sriov-net
           sriov: {}
-   resources:
-     limits:
-       intel.com/sriov: "1"
-     requests:
-       intel.com/sriov: "1"
+    resources:
```
Comment (rmohr) [inline workloads/virtual-machines/interfaces-and-networks.md]: Hm, I fear that there is one whitespace more required. Can that be?

Comment (booxter) [reply workloads/virtual-machines/interfaces-and-networks.md]: @rmohr I've checked locally, it seems to consistently use 2-space indent for all levels.

Comment (rmohr) [reply workloads/virtual-machines/interfaces-and-networks.md]: I see. My mistake. Good to go then. :+1:

**Pull request merged**